llvm/CMakeLists.txt

```diff
@@ -552,6 +552,10 @@ else()
   option(LLVM_ENABLE_THREADS "Use threads if available." ON)
 endif()
 
+set(LLVM_ENABLE_ICU "ON" CACHE STRING "Use ICU for character conversion support if available. Can be ON, OFF, or FORCE_ON")
```

Comment:
These should default to OFF.

Reply:
> These should default to OFF.

Both? The `iconv` support used is already limited to "built-in" support, e.g., from `libc`.